tools/pylib/_boutcore_build/boutcore.pyx.in

```diff
@@ -462,6 +487,390 @@ cat <<"EOF"
             del self.cobj
             self.cobj=NULL
 
+cdef Field2D f2dFromObj(c.Field2D i):
```

Comment:
I guess this is copy pasted from the Field3D?
If so, can this be replaced by a for loop over all Fields?
This will make it easier to keep the interface aligned, and easier to fix bugs, like the raise of a string, rather an exception ...